test/autorest/optionalgroup/zz_generated_explicit_client.go

```diff
@@ -341,28 +341,28 @@ func (client *ExplicitClient) postOptionalStringHeaderCreateRequest(ctx context.
 	return req, nil
 }
 
-// PostOptionalStringParameter - Test explicitly optional string. Please put null.
+// PostOptionalStringParameterWithText - Test explicitly optional string. Please put null.
```

Comment:
This feature should be off by default.  We shouldn't be changing any existing names.

Reply:
Change default to `false`.